app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] [OPTIONAL] Please modify your login and registration form views to match the layout of the wireframes 👍 
### Wireframe
![image](https://github.com/Gmanlove/blog-app/assets/46839548/75a48554-98f6-4804-80f7-5a148a326ecf)#Gmanlove 
### Your Implementation
![image](https://github.com/Gmanlove/blog-app/assets/46839548/63641c07-50a2-486b-9678-fec81427362d)